pkg/ebpf/c/common/buffer.h

```diff
@@ -43,7 +44,7 @@ statfunc data_filter_lpm_key_t *get_string_data_filter_lpm_buf(int idx)
 }
 
 // biggest elem to be saved with 'save_to_submit_buf' should be defined here:
-#define MAX_ELEMENT_SIZE bpf_core_type_size(struct sockaddr_un)
+#define MAX_ELEMENT_SIZE sizeof(struct sockaddr_un)
```

Comment:
Why this change?

Reply:
Probably a verifier issue at some point. I will check while rebasing if I can revert this.